File: GymWare-Backend/GymWare.Entities/EmpleadoClienteRutina.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GymWare.Entities
{
    public class EmpleadoClienteRutina
    {
        public int EmpleadoClienteRutinaId { get; set; }
        public virtual Cliente Cliente { get; set; }
        public virtual Empleado Empleado { get; set; }
        public virtual Rutina Rutina { get; set; }
        [Column(TypeName = "date")]
        public DateTime FechaInicio { get; set; }
        [Column(TypeName = "date")]
        public DateTime FechaFin { get; set; }
    }
}

File: GymWare-Frontend/src/app/classes/EmpleadoClienteRutina.ts
import { Cliente } from "./Cliente";
import { Empleado } from "./Empleado";
import { Rutina } from "./rutina";

export class EmpleadoClienteRutina {

    EmpleadoClienteRutinaId: number;
    Cliente: Cliente;
    Empleado: Empleado;
    Rutina: Rutina;
    FechaInicio: String;
    FechaFin: String;
}